<!-- Launch file for the rrt-detector and the assigner -->

<launch>
  <node pkg="rrt_exploration" type="assigner.py" name="assigner" output="screen">
  <!--param name="map_topic" value="/map_merge/map"/-->
  <param name="map_topic" value="/robot_1/map"/>
  <param name="info_radius" value="1"/> 
  <param name="info_multiplier" value="3.0"/> 
  <param name="hysteresis_radius" value="3.0"/> 
  <param name="hysteresis_gain" value="2.0"/> 
  <param name="goals_topic" value="/exploration_goals"/> 
  <param name="n_robots" value="1"/>
  </node>
</launch>
